registry/server/util/logger.ts

```diff
@@ -0,0 +1,52 @@
+import { Logger } from 'ilc-plugins-sdk';
+import { getPluginManagerInstance } from './pluginManager';
+import { storage } from '../middleware/context';
+
+const hasKey = <T extends object>(obj: T, k: keyof any): k is keyof T => k in obj;
+
+const loggerProxyHandler: ProxyHandler<Logger> = {
+    get(target, prop) {
+        const requestIdKey = getPluginManagerInstance().getReportingPlugin().requestIdLogLabel ?? 'operationId';
+        const origMethod = hasKey(target, prop) && target[prop];
+        if (typeof origMethod === 'function') {
```

Comment:
could you please invert condition to reduce nesting. And probably in order not to break to much, the `else` case should be handled as well